src/convertToObject.js

```diff
@@ -6,7 +6,18 @@
  * @return {object}
  */
 function convertToObject(sourceString) {
-  // write your code here
+  const resObj = {};
+  const arrayWithoutSpaces = sourceString.trim().split(';');
+
+  arrayWithoutSpaces.forEach((item) => {
+    const [key, value] = item.split(':').map((str) => str.trim());
+
+    if (key && value) {
+      resObj[key] = value;
+    }
+  });
```

Comment:
[CODE PRACTICE] - Avoid using loops in this task, use iteration methods instead. However, you have correctly used `forEach` which is an exception to this rule. Good job on that part.